### DISCUSSION TITLE
Week 4

### DISCUSSION THREAD

**Discussion by DavidDHBW** [DavidDHBW/codespaces-jupyter#6] [General]
Polished and finished the hangman Project. Fixed Bugs. 
Start of a new project: a chess game. Thought of how to start this properly. Realized the board with an field of strings.
[code](https://github.com/DavidDHBW/codespaces-jupyter/tree/main/Python)

**Comment by LxisW** [comment]
Cool idea to start with an easy game like hangman and now go on with something more difficult.

**Comment by emmanuelyoussefyoussef** [comment]
just thinking about your new project and it sounds very complicated iam looking for your game👌
